coursera.py

```diff
@@ -0,0 +1,41 @@
+import os
+import shutil
+import sys
+
+def check_reboot():
+    '''Returns True if the computer has a pending reboot'''
+    return os.path.exists('/run/reboot-required')
+
+def check_disk_full(disk, min_gb, min_percent):
```

Comment:
Your function checks disk space and return true or false, but regarding function name it says to check only full, so I would call it checkDiskSpace